uint32 id
uint32 row
uint32 col
geometry_msgs/Point location_odom
geometry_msgs/Point location_base_link